scripts/test_pk_enc.py

```diff
@@ -0,0 +1,99 @@
+import os
```

Comment:
This file shouldn't be here. If you want to test it, do it externally or fork the python bfv dependency and add a test there